src/pages/AppList.svelte

```diff
@@ -30,9 +31,10 @@
   <FilterInput onChangeText={filterApps} />
   {#each filteredApps as app}
     {#if showDeprecated || !app.deprecated}
-      <p>
+      <p class="mb-2">
         <a href="/apps/{app.name}">{app.name}</a>
         {#if app.description}<i>{app.description}</i>{/if}
+        <MetaData data={app} />
```

Comment:
As above, this should only display if the deprecated property is true. Also we should just show "Deprecated", not "Deprecated: true".